packages/grid/src/vaadin-grid-drag-and-drop-mixin.js

```diff
@@ -103,6 +103,11 @@ export const DragAndDropMixin = (superClass) =>
         __dndAutoScrollThreshold: {
           value: 50,
         },
+
+        /** @private  */
+        __draggedItems: {
+          value: [],
```

Comment:
When using a non-primitive for property `value` a function that returns the value must be used, see https://polymer-library.polymer-project.org/3.0/docs/devguide/properties#configure-values

Could also consider not declaring the private property at all in the `properties` listing.